libs/scripts/scripts.js

```diff
@@ -123,7 +123,8 @@ const config = {
   geoRouting: 'on',
   fallbackRouting: 'on',
   links: 'on',
-  imsClientId: 'milo',
+  // will undo before merging
+  imsClientId: new URLSearchParams(window.location.search).get('imsClientId') || 'milo', // TODO: reset to original value
```

Comment:
We should either keep this or start adding a few branches to the milo token.

Reply:
Yeah, this should be removed, I used it for a another branch for testing purposes, but as the TODO says, it shouldn't be merged in